### DISCUSSION TITLE
GRID大作业中关于时间轴是否必然等距的提问？

### DISCUSSION THREAD

**Discussion by DM0U** [physics-data/faq#137] [Q&A] [answered]
在GRID大作业的样例source.h5文件中，观测发现所有相邻的time之间恒定差为0.064s，想问一下这是巧合还是说最终的数据也能满足等距取点的条件？

（当然理论上编程的时候可以默认安装非等时间差假设进行求和，但我想知道这是否意味着可以用平均操作来代替加权求和，并且必然不会出错？

**Comment by heroxbd** [comment]
@babychen9816

**Comment by quxnb** [comment, marked as answer]
source.h5 中时间等距处理只是为了方便, 也可以设置为不等距, 但需要在报告中写明你的做法以及这样做的合理性.
